app/forms/declaration/officeOfExit/OfficeOfExit.scala

```diff
@@ -17,22 +17,18 @@
 package forms.declaration.officeOfExit
 import play.api.libs.json.Json
 
-case class OfficeOfExit(officeId: Option[String], circumstancesCode: Option[String])
+case class OfficeOfExit(officeId: Option[String], isUkOfficeOfExit: Option[String])
```

Comment:
I like this.  Did you say that "circumstancesCode" has not been used for a while?

Reply:
No. It was being used until this change.